app/views/home/about.html.erb

```diff
@@ -15,5 +15,5 @@ commercial exploitation and legal challenge.") %></li>
 that seeks to promote scholarship on fanworks and practices.") %></li>
     </ul>
   </p>
-  <p><%= ts("You can find out more about the OTW and its projects at its website,") %> <%= link_to ts("transformativeworks.org"), "http://transformativeworks.org" %><%= ts(", and learn about how your financial support is vital to the continuation and expansion of the OTW's work on its") %> <%= link_to ts("FAQ page"), "http://transformativeworks.org/faq-page" %>.</p>
+  <p><%= ts("You can find out more about the OTW and its projects at its website,") %> <%= link_to ts("transformativeworks.org"), "http://transformativeworks.org" %><%= ts(", and learn about how your financial support is vital to the continuation and expansion of the OTW's work on its") %> <%= link_to ts("FAQ page"), "http://transformativeworks.org/faq" %><%= ts(". If you have a media or research question, please contact the") %> <%= link_to ts("Communications team"), "http://transformativeworks.org/contact_us/concomm" %>.</p>
```

Comment:
i don't mind making these changes, and it's good practice since i'm still not used to working with ruby~